tests/framework-tests/preferences/js/SeparatedPanelPrefsEditorResponsiveTests.js

```diff
@@ -0,0 +1,337 @@
+/*
+Copyright 2017 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid, jqUnit */
+
+
+(function ($) {
+    "use strict";
+
+    fluid.registerNamespace("fluid.tests");
+
+    /******************
+     * Setup sequence *
+     ******************/
+
+    fluid.defaults("fluid.tests.iframeSetup", {
+        gradeNames: "fluid.test.sequenceElement",
+        events: {
+            iframeLoaded: null
+        },
+        iframe: ".flc-iframeSetup",
+        sequence: [{
+            func: "fluid.tests.iframeSetup.load",
+            args: ["{that}"]
+        }]
+    });
+
+    fluid.tests.iframeSetup.load = function (that) {
```

Comment:
Please put these definitions in a more descriptive namespace - since they are hardwired to start a pageEnhancer test. The name should at least indicate what broad area of the framework is being tests, e.g. fluid.tests.prefs or so